server/utils/jsonApi.js

```diff
@@ -72,39 +72,45 @@ const buildSecondaryTags = arr =>
   }));
 
 const HUB_CONTENT_TYPES = {
-  moj_radio_item: {
+  moj_radio_item: () => ({
     contentType: 'radio',
     externalContent: false,
-  },
-  moj_video_item: {
+  }),
+  moj_video_item: () => ({
     contentType: 'video',
     externalContent: false,
-  },
-  moj_pdf_item: {
+  }),
+  moj_pdf_item: () => ({
     contentType: 'pdf',
     externalContent: true,
-  },
-  external_link: {
-    contentType: 'external_link',
-    externalContent: true,
-  },
-  page: {
+  }),
+  link: item =>
+    item?.fieldShowInterstitialPage === true
```

Comment:
little bit gnarly - this is because we represent both types of link with the same content type. 

We've agreed as its a very small minor case its not worth effort of maintaining additional content type in drupal. The added complexity here isn't a huge amount